examples/image-classification-web/index.html

```diff
@@ -0,0 +1,243 @@
+      integrity="sha256-9+AQR2dApXE+f/D998vy0RATN/o4++mqVjAZ3lo432g="
+      crossorigin="anonymous"
+    ></script>
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/chart.js@4.2.1/dist/chart.umd.min.js"
+      integrity="sha256-tgiW1vJqfIKxE0F2uVvsXbgUlTyrhPMY/sm30hh/Sxc="
+      crossorigin="anonymous"
+    ></script>
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/chartjs-plugin-datalabels@2.2.0/dist/chartjs-plugin-datalabels.min.js"
+      integrity="sha256-IMCPPZxtLvdt9tam8RJ8ABMzn+Mq3SQiInbDmMYwjDg="
+      crossorigin="anonymous"
+    ></script>
+
+    <script src="./index.js"></script>
+
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/normalize.min.css@8.0.1/normalize.min.css"
+      integrity="sha256-oeib74n7OcB5VoyaI+aGxJKkNEdyxYjd2m3fi/3gKls="
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="./index.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="selections">
+        <!-- Backend Selection -->
+        <div class="select-box">
+          1.
+          <label for="backend">Backend:</label>
+          <select id="backend">
+            <option value="ndarray" selected>CPU - Ndarray</option>
+            <option value="candle">CPU - Candle</option>
+            <option value="webgpu">GPU - WebGPU</option>
+          </select>
+        </div>
+      </div>
+
+      <div class="row-container">
+        <!-- Image Selection -->
+        <div class="select-box">
+          <!-- <label for="imageDropdown">Select Image:</label> -->
+          2.
+          <select id="imageDropdown">
+            <option value="" selected>Select Image</option>
+            <option value="samples/bridge.jpg">Bridge</option>
+            <option value="samples/cat.jpg">Cat</option>
+            <option value="samples/coyote.jpg">Coyote</option>
+            <option value="samples/flamingo.jpg">Flamingo</option>
+            <option value="samples/pelican.jpg">Pelican</option>
+            <option value="samples/table-lamp.jpg">Table Lamp</option>
+            <option value="samples/torch.jpg">Torch</option>
+          </select>
+          or
+          <!-- <label for="fileInput">Upload an Image:</label> -->
+          <input type="file" id="fileInput" accept="image/*" />
+        </div>
+      </div>
+
+      <!-- Time Taken -->
+      <div id="time">&nbsp;</div>
+
+      <!-- Container for the three boxes -->
+      <div class="row-container">
+        <!-- Canvas to Display Image -->
+        <div class="canvas-box">
+          <canvas id="imageCanvas" width="224" height="224"></canvas>
+        </div>
+
+        <!-- Chart -->
+        <div class="chart-box">
+          <canvas id="chart" width="500" height="224"></canvas>
+        </div>
+      </div>
+
+      <!-- Clear Button -->
+      <div class="actions">
+        <button id="clearButton">Clear</button>
+      </div>
+    </div>
+
+    <!-- JavaScript Logic -->
+    <script type="module">
```

Comment:
Could we put this logic into `index.js` separating html from js.

Reply:
I tried but it requires more thought and refactoring. If it's okay, lets leave as it is for now. On next update, I'll do refactoring among xtask.

Reply:
I added a todo refactor